pages/fundamentals/constraints.mdx

```diff
@@ -161,6 +161,102 @@ DROP CONSTRAINT ON (n:Employee) ASSERT n.name, n.address IS UNIQUE;
 
 Now, `SHOW CONSTRAINT INFO;` returns an empty set.
 
+## Data type constraint
+
+The data type constraint enforces that each label-property pair is of a certain data type.

```

Comment:
There's a corner case I'm not seeing addressed here. What if two different labels apply different constraints to the same property?

CREATE (n:Circle:Blue {name: "John Doe", age: 30});
CREATE CONSTRAINT ON (n:CIrcle) ASSERT n.age IS TYPED integer;
CREATE CONSTRAINT ON (n:Blue) ASSERT n.age IS TYPED float;

What happens here?

Reply:
Covered below by
`You can only have one data type constraint on a given label-property pair.`

Reply:
Sorry, I'm confused. What happens when I try to store 5.5 into the `age` property? Does it work or does it violate the constraint?

This is not just about "can the constraint exist" - it is also about how does the constraint behave later in time during normal usage.

My mental model is that each node has a single property store. Is that correct?

Or is there an extra property store for each label on the node? So a node with 5 labels would have 5 property stores?